web-console/src/druid-models/lookup-spec.tsx

```diff
@@ -268,22 +268,6 @@ export const LOOKUP_FIELDS: Field<LookupSpec>[] = [
   },
 
   // JDBC stuff
-  {
```

Comment:
From looking at Travis failure, It looks like this requires a snapshot update. `jest --updateSnapshot`

Reply:
Thanks for the review. I reverted this change in favor of #10888.